pyproject.toml

```diff
@@ -1,10 +1,8 @@
 [build-system]
-requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
+requires = ["setuptools>=61.0"]
```

Comment:
Revert to be consistent with https://packaging.python.org/en/latest/guides/writing-pyproject-toml/.

Reply:
rebased onto master.